src/applications/mhv-landing-page/tests/e2e/mhv-auth-redirect.cypress.spec.js

```diff
@@ -16,7 +16,7 @@ describe(`${appName} -- Auth Redirect`, () => {
 
   describe('unauthenticated user', () => {
     // eslint-disable-next-line @department-of-veterans-affairs/axe-check-required
-    it('prompts user to authenticate, redirecting to /my-health', () => {
+    it.skip('prompts user to authenticate, redirecting to /my-health', () => {
```

Comment:
Remove/refactor/re-enable test if it will stay skipped.

Reply:
This test is failing in CI on [multiple branches](https://github.com/department-of-veterans-affairs/vets-website/pull/27319#commits-pushed-2bc3b95). I created a ticket to investigate this issue. department-of-veterans-affairs/va.gov-team#72838

Reply:
going to keep this as is and save the fixing of this for the new ticket

Reply:
Link the new ticket here for tracking.

Reply:
department-of-veterans-affairs/va.gov-team#72838